From 2c4c813940c577590f3352cef0c49a8def17905d Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@bootlin.com>
Date: Wed, 28 Apr 2021 22:58:45 +0200
Subject: [PATCH] include/configs/starfive-vic7100: adjust fdt_addr_r

The default fdt_addr_r of 0x88000000 doesn't work, the kernel never
boots. Using 0x90000000 works fine.

Since it would overlap with the kernel_comp_addr_r area, this one is
moved 16 MB further, at 0x91000000.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@bootlin.com>
---
 include/configs/starfive-vic7100.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/configs/starfive-vic7100.h b/include/configs/starfive-vic7100.h
index 8c5915a73c..7150a23873 100644
--- a/include/configs/starfive-vic7100.h
+++ b/include/configs/starfive-vic7100.h
@@ -111,13 +111,13 @@
 	"fdt_high=0xffffffffffffffff\0" \
 	"initrd_high=0xffffffffffffffff\0" \
 	"kernel_addr_r=0x84000000\0" \
-	"fdt_addr_r=0x88000000\0" \
+	"fdt_addr_r=0x90000000\0" \
 	"scriptaddr=0x88100000\0" \
 	"script_offset_f=0x1fff000\0" \
 	"script_size_f=0x1000\0" \
 	"pxefile_addr_r=0x88200000\0" \
 	"ramdisk_addr_r=0x88300000\0" \
-	"kernel_comp_addr_r=0x90000000\0" \
+	"kernel_comp_addr_r=0x91000000\0" \
 	"kernel_comp_size=0x10000000\0" \
 	"type_guid_gpt_loader1=" TYPE_GUID_LOADER1 "\0" \
 	"type_guid_gpt_loader2=" TYPE_GUID_LOADER2 "\0" \
-- 
2.30.2

